src/components/leaderBoard.js

```diff
@@ -20,12 +20,12 @@ const Scores = (props) => {
         return  (
             props.scores.map((score,  i) => {
                 return (
-                    <ListGroupItem key={i}>
```

Comment:
Need to remove import of ListGroupItem and ListGroup from the top of the file since we're not using them anymore.

Reply:
fixed